README.md

```diff
@@ -2,16 +2,76 @@
 
 [![License](https://img.shields.io/badge/License-Apache-blue.svg)](LICENSE)
 
-## Description
+# Realm Dart Samples
 
-This repo contains Realm Flutter and Realm Dart SDK samples. 
+This repo contains Flutter and Dart samples that demonstate the usage of [Realm Flutter and Dart Standalone SDKs](https://www.mongodb.com/docs/realm/sdk/flutter/).
 
-The Realm Flutter and Realm Dart SDK are located at https://github.com/realm/realm-dart
+To build your own app with Realm, use the [realm](https://pub.dev/packages/realm) package for Flutter and the [realm_dart](https://pub.dev/packages/realm_dart) package for Dart applications.
 
+The Realm Flutter and Realm Dart SDK repository is located at https://github.com/realm/realm-dart
 
-## Usage 
-Check the README.md files of the samples for instructions how to run them.
+You may find the [Quick Start - Flutter SDK]( https://www.mongodb.com/docs/realm/sdk/flutter/quick-start/) helpful if you are using Realm for the first time.
+To learn more about using Realm with Atlas App Services, refer to the [Atlas Device Sync](https://www.mongodb.com/docs/realm/sdk/flutter/sync/) and [connecting to Atlas App Services](https://www.mongodb.com/docs/realm/sdk/flutter/app-services/) documentation.
 
+## [`realm_dart`](https://github.com/realm/realm-dart-samples/tree/main/realm_dart) - Dart console application
```

Comment:
```suggestion
### [`realm_dart`](https://github.com/realm/realm-dart-samples/tree/main/realm_dart) - Dart console application
```

Reply:
Not sure but my suggestion is to remove the underline on all the samples. Also converting this to a bullet list will so
* realm_dart - Dart console app demonstrates how to use a local Realm.
* `Provider Shopper` - A Flutter sample app from [Flutter sample repo](https://github.com/flutter/samples/tree/master/provider_shopper) that shows a state management approach with a local Realm.
 * [time_track](https://github.com/realm/realm-dart-samples/tree/main/time_track) - Dart console time tracking application
 * [dart_flexible_sync](https://github.com/realm/realm-dart-samples/tree/main/dart_flexible_sync) - This sample demonstrates the usage of Realm Dart with [Atlas Device Sync with Flexible Sync](https://www.mongodb.com/docs/realm/sdk/flutter/sync/) sync data to Atlas App Services. (link needed)
 * [flutter_flexible_sync](https://github.com/realm/realm-dart-samples/tree/main/flutter_flexible_sync) - A Flutter application using the [Realm Flutter SDK](https://www.mongodb.com/docs/realm/sdk/flutter/), Atlas Device Sync with Flexible Sync, and [Atlas App Services](https://www.mongodb.com/docs/atlas/app-services/).
 * [bundled_realm](https://github.com/realm/realm-dart-samples/tree/main/bundled_realm) - A Flutter application with pre-initialized realm file that is shipped with the application.


Also the names of the samples listed here should not follow the directory name but rather the sample name. So lets use that instead. 
* Realm Dart sample
* Dart Flexible Sync sample
* Flutter Flexible Sync sample
* Bundled realm sample

Reply:
Done.